aspnet/fundamentals/localization.rst

```diff
@@ -204,6 +204,9 @@ CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
```

Comment:
@JunTaoLuo If I quite understand, the reference to ASPNET_CULTURE  was set by team just to compare with "old way". May be I get it wrong.